### DISCUSSION TITLE
composer install for php

### DISCUSSION THREAD

**Discussion by areaboy** [vercel/community#3080] [Help]
### Summary

I need help please.

I have some php file that requires installation of files via composer

**myproject/api/app.php**

How do I run or install this 2 file via composer for php app on vercel

**composer require aws/aws-sdk-php**

**composer require phpmailer/phpmailer**

**Here is my vercel.json**

```

{
  "functions": {
    "api/*.php": {
      "runtime": "vercel-php@0.6.0"
    }
  },
  "routes": [
    { "src": "/api/(.*)",  "dest": "/index.php" }
  ]
}


```

### Steps to Reproduce

Thanks

**Comment by pife00** [comment]
I think I face the same problem I use https://github.com/juicyfx/vercel-examples composer, at first I've got some errors because composer locker doesn't applied some change from head, but everything right until I install a new package, and I got composer.json and composer.locker are not syncs, so composer update and doesn't work, but if you go at this branch in vercel you can re deploy and works, look likes vercel doesn't update composer.lock when sync with github
